include/lsst/afw/formatters/Utils.h

```diff
@@ -139,7 +141,24 @@ int extractVisitId(std::shared_ptr<lsst::daf::base::PropertySet const> const& pr
 int extractCcdId(std::shared_ptr<lsst::daf::base::PropertySet const> const& properties);
 int extractAmpId(std::shared_ptr<lsst::daf::base::PropertySet const> const& properties);
 
+/**
+Format a PropertySet into a single FITS header string
+
+@param[in] prop  Properties to format
+@return a FITS header string (exactly 80 characters per "card", no line terminators)
```

Comment:
I'd prefer getting a list of cards, so I can look at each individually or print it and have it make sense if my terminal isn't set to width=80. Concatenating them is trivial (`sum(cards, "")`), but splitting a single string is less trivial (not difficult, but annoying).